docs_src/walk-through/Ch-WalkthroughExporting.md

```diff
@@ -49,7 +59,7 @@ On the Key/Value display page, click on `edgex` > `appservices` > `1.0` > `AppSe
 The log level change will be picked up by the application service.  In a terminal window, execute the Docker command below to view the service log.
 
 ``` shell
-docker logs -f edgex-app-service-configurable-rules
+docker logs -f edgex-app-rules-engine
```

Comment:
Note the image below is out of date. The debug log messages have changed. It is a bit more verbose... See attached image.
![image](https://user-images.githubusercontent.com/44779287/132915008-b77eeef5-0a45-4a1f-a879-c12fb55858af.png)